.github/workflows/publish-olm-bundle.yaml

```diff
@@ -0,0 +1,80 @@
+name: Submit release OLM bundle
+
+on:
+  release:
+    types:
+      - published
+# Remove this trigger event once workflow behavior is validated
+  issues:
+    types:
+      - opened
+
+jobs:
+  createPullRequest:
+    name: Publish new OperatorHub release
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Determine Repository Variable
+      # Remove this conditional logic once the workflow behavior
+      # is validated
+          run: |
```

Comment:
This looks to  me like an incorrect indentation of `run`.

Reply:
fixed